utils/getTrustedCertificates.ts

```diff
@@ -0,0 +1,107 @@
+/*---------------------------------------------------------------------------------------------
+ *  Copyright (c) Microsoft Corporation. All rights reserved.
+ *  Licensed under the MIT License. See LICENSE.md in the project root for license information.
+ *--------------------------------------------------------------------------------------------*/
+
+import * as fse from 'fs-extra';
+import * as https from 'https';
+import * as path from 'path';
+import * as vscode from 'vscode';
+import { callWithTelemetryAndErrorHandling, IActionContext } from "vscode-azureextensionui";
+import { ext } from '../extensionVariables';
+import { globAsync } from '../helpers/async';
+import { isLinux, isMac, isWindows } from '../helpers/osVersion';
+
+let _systemCertificates: (string | Buffer)[] | undefined;
+
+export async function getTrustedCertificates(): Promise<(string | Buffer)[]> {
+    // tslint:disable-next-line:no-function-expression
+    return callWithTelemetryAndErrorHandling('docker.certificates', async function (this: IActionContext): Promise<(string | Buffer)[]> {
+        this.suppressTelemetry = true;
+
+        let systemCerts = getCertificatesFromSystem();
+
+        let certificatePaths = vscode.workspace.getConfiguration('docker').get<string[] | undefined>('certificates');
```

Comment:
If this is undefined - can't we default to the behavior before your PR? Why do we need to specify any ca stuff?

Reply:
That would mean Linux wouldn't work out of the box.  This way it should work most of the time.

Reply:
Not sure why we would want that.

Reply:
Never mind - I thought it was already reading from system store before your PR